lib/readwise/tag.rb

```diff
@@ -0,0 +1,7 @@
+module Readwise
+  Tag = Struct.new(
+    'ReadwiseTag',
+    :name,
+    keyword_init: true
+  )
+end
```

Comment:
Here's how to avoid the "No newline at end of file" warning:

https://thoughtbot.com/blog/no-newline-at-end-of-file